index.html

```diff
@@ -600,6 +600,37 @@ <h4>linkedResource</h4>
     }</pre>
       
       </section>
+
+        <section>
+          <h4>dnsDomain</h4>
+          <table class="simple" style="width: 100%;">
+            <thead>
+              <tr>
+                <th>Normative Definition</th>
+                <th>JSON-LD</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <a
+                    href="https://www.ietf.org/archive/id/draft-carter-high-assurance-dids-with-dns-04.html#name-the-dnsdomain-property">High Assurance DIDs with DNS</a>
+                </td>
+                <td>
+                  <a href="https://ciralabs.github.io/DnsDomainJsonLd/dns-domain.jsonld">dnsDomain JSON-LD
+                    Context</a>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        
+          <pre class="example" title="Example of dnsDomain property">
+{  
+  "dnsDomain": "example.com"
```

Comment:
Domain examples are supposed to use ".example" these days (example.com is historical usage, IIRC).

```suggestion
  "dnsDomain": "mydomain.example"
```